kasa/smartcamera/modules/camera.py

```diff
@@ -38,6 +40,24 @@ def is_on(self) -> bool:
         """Return the device id."""
         return self.data["lens_mask_info"]["enabled"] == "off"
 
+    def _get_credentials(self) -> Credentials | None:
+        """Get credentials from ."""
+        config = self._device.config
+        if credentials := config.credentials:
+            return credentials
+
+        if credentials_hash := config.credentials_hash:
+            try:
+                decoded = json_loads(
+                    base64.b64decode(credentials_hash.encode()).decode()
+                )
+            except Exception:
+                return None
```

Comment:
Should we log this / have some error handling?

Reply:
Agreed, that's done